pom.xml

```diff
@@ -16,8 +16,15 @@
     <properties>
         <java.version>17</java.version>
         <maven.checkstyle.plugin.configLocation>checkstyle.xml</maven.checkstyle.plugin.configLocation>
+        <lombok.mapstruct.binding.version>0.2.0</lombok.mapstruct.binding.version>
+        <mapstruct.version>1.5.5.Final</mapstruct.version>
     </properties>
     <dependencies>
+        <dependency>
+            <groupId>org.liquibase</groupId>
+            <artifactId>liquibase-core</artifactId>
+            <version>4.24.0</version>
```

Comment:
please double-check whether this dependency is not present in `spring-boot-starter-parent` (If I'm not mistaken - it should be there)

if it is there - you don't need to specify the version in your pom.xml